dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphHostControl.xaml.cs

```diff
@@ -42,16 +41,16 @@ namespace VisualLayerIntegration
     /// </summary>
     public partial class BarGraphHostControl : UserControl
     {
-        private CompositionHost compositionHost;
-        private Compositor compositor;
-        private Windows.UI.Composition.ContainerVisual graphContainer;
-        private BarGraph currentGraph;
+        private CompositionHost _compositionHost;
```

Comment:
Now that the initialization is in the constructor, most of these can be made readonly. Make as much readonly as possible.

Reply:
updated